index.html

```diff
@@ -1,43 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="author" content="Keri West" />
-    <meta name="description" content="Keri West's website is a digital portfolio to display his coding work and a little about himself."/>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Keri West">
+    <meta name="description" content="Keri West's website is a digital portfolio to display his coding work and a little about himself.">
 
     <title>Keri West's Website</title>
 
-    <link rel="stylesheet" type="text/css" href="css/index.css" />
-    <link href="https://fonts.googleapis.com/css2?family=Belanosima:wght@600&family=Roboto+Mono:wght@300;400&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" type="text/css" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Belanosima:wght@600&family=Roboto+Mono:wght@300;400&display=swap" rel="stylesheet">
   </head>
 
   <body>
-    <div id="navbar">
-        <a href="#about">About_me</a></li>
-        <a href="#experience">My_experience</a></li>
-        <a href="#form">Let's_connect</a></li>
-    </div>
+    <div>
+      <nav>
+        <ul class="nav-list">
+          <li class="icon"><a href="#icon"><img src="img/icons/KWEST_Logo_v01.svg" alt="kwest site icon"></a></li>
```

Comment:
```suggestion
          <li class="icon"><img src="img/icons/KWEST_Logo_v01.svg" alt="kwest site icon"></li>
```

Doesn't really go anywhere. Consider removing `<a>`